src/themes/ec/variables/_button.scss

```diff
@@ -3,11 +3,13 @@
 
 $button: (
   radius: 2px,
+  font: var(--f-m),
   shadow: none,
   border-width: 2px,
   outline-width: 1px,
   padding-horizontal: var(--s-m),
   padding-vertical: var(--s-xs),
+  icon-only-size: map.get($icon, 'xs'),
```

Comment:
i wonder why we are notsimply doing the same in EC and EU, we aligned now the default icon size but we introduced this difference in the icon only use case..

Reply:
I'll check with the designers